src/project/FileTreeView.js

```diff
@@ -478,7 +478,42 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
```

Comment:
@humphd   I was also surprised. At the first time, I add `. `to all the cases, and my icons disappeared.
By the way, I added some cases, please review it. @flukeout